commands/cloudapi-v6/s3key.go

```diff
@@ -252,23 +260,71 @@ func RunUserS3KeyUpdate(c *core.CommandConfig) error {
 }
 
 func RunUserS3KeyDelete(c *core.CommandConfig) error {
-	if err := utils.AskForConfirm(c.Stdin, c.Printer, "delete s3key"); err != nil {
-		return err
-	}
-	c.Printer.Verbose("User ID: %v", viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgUserId)))
-	c.Printer.Verbose("Deleting S3 Key with ID: %v...", viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgS3KeyId)))
-	resp, err := c.CloudApiV6Services.S3Keys().Delete(viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgUserId)),
-		viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgS3KeyId)),
-	)
-	if resp != nil {
-		c.Printer.Verbose(cloudapiv6.RequestTimeMessage, resp.RequestTime)
+	var resp *resources.Response
+	userId := viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgUserId))
+	s3KeyId := viper.GetString(core.GetFlagName(c.NS, cloudapiv6.ArgS3KeyId))
+	allFlag := viper.GetBool(core.GetFlagName(c.NS, cloudapiv6.ArgAll))
+	if allFlag {
+		err := DeleteAllS3keys(c)
```

Comment:
Here, it needs to be returned also the `resp`, and not use ":=" to not define it as a new variable as it is already defined above and used outside the if loop.